browser/ui/webui/welcome_page/welcome_dom_handler.cc

```diff
@@ -173,6 +183,37 @@ void WelcomeDOMHandler::HandleEnableWebDiscovery(
   profile_->GetPrefs()->SetBoolean(kWebDiscoveryEnabled, true);
 }
 
+void WelcomeDOMHandler::HandleGetWelcomeCompleteURL(
+    const base::Value::List& args) {
+  CHECK_EQ(1U, args.size());
+  const auto& callback_id = args[0].GetString();
+  if (!base::FeatureList::IsEnabled(
+          brave_education::features::kShowGettingStartedPage)) {
+    OnGettingStatedServerCheck(callback_id, /* available */ false);
```

Comment:
should be OnGettingStartedServerCheck ?

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/6313ac781a12164601ca4af8668f08ca31a4860e